source/NVDAObjects/IAccessible/__init__.py

```diff
@@ -392,7 +392,7 @@ def findOverlayClasses(self,clsList):
 			clsList.insert(0, FocusableUnfocusableContainer)
 
 		if hasattr(self, "IAccessibleTextObject"):
-			if role==oleacc.ROLE_SYSTEM_TEXT or self.IA2States&IAccessibleHandler.IA2_STATE_EDITABLE:
+			if role==oleacc.ROLE_SYSTEM_TEXT or controlTypes.STATE_EDITABLE in self.states:

```

Comment:
I'm a bit bothered about this one. Are you sure that this won't cause side effects in totally unrelated applications?